<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Keyboard</class>
 <widget class="QWidget" name="Keyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>keyboard</string>
  </property>
  <widget class="QLineEdit" name="display">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>10</y>
     <width>400</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>800</width>
     <height>171</height>
    </rect>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>10</y>
      <width>16</width>
      <height>156</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_z">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_n">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>n</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_8">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_i">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_j">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>j</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_u">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>u</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_w">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>w</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_inv">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>124</y>
      <width>66</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Inv</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_c">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_xinghao">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_line1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>46</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_fenhao">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>85</y>
      <width>85</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_p">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_e">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_esc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>124</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Clr</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_9">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_t">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_0">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_s">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_r">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_h">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_a">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_y">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_l">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>l</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_f">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>f</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_line3">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>85</y>
      <width>66</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_right">
    <property name="geometry">
     <rect>
      <x>499</x>
      <y>124</y>
      <width>77</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>--&gt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_q">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>q</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_b">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_o">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_left">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>124</y>
      <width>77</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_v">
    <property name="geometry">
     <rect>
      <x>291</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_capslock">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string>Caps</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_k">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>k</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_wenhao">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_x">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_backspace">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>124</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>DEL</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_enter">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>124</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_7">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>7</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_dot">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_d">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>d</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_space">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>124</y>
      <width>281</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Space</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_g">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>46</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_line2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>65</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>\</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_m">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>85</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Input:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>display</cstring>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>display</tabstop>
  <tabstop>toolButton_capslock</tabstop>
  <tabstop>toolButton_esc</tabstop>
  <tabstop>toolButton_inv</tabstop>
  <tabstop>toolButton_left</tabstop>
  <tabstop>toolButton_right</tabstop>
  <tabstop>toolButton_backspace</tabstop>
  <tabstop>toolButton_enter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
